### ISSUE TITLE
Include how to start the tool in the readme

### ISSUE THREAD

**Issue by karlnapf** [shogun-toolbox/applications#2]
@Hephaestus12 this would be good to have

**Comment by karlnapf** [comment]
* how to start the tool
* link to your gsoc blog posts

**Comment by Hephaestus12** [comment]
I'll link the blog posts.

I have instructions to run it here: https://github.com/shogun-toolbox/applications/blob/master/gsoc_application_projects/2020/influenza/web/README.md

Should I add it to the root README.md file too?

**Comment by karlnapf** [comment]
absolutely. Or rather make a link "Instruction to run the web-app" that points to that file.
Just pretending I am a person coming by this for the first time ...
